addons/web/static/src/webclient/webclient_layout.scss

```diff
@@ -34,7 +34,7 @@ html {
         > .o_control_panel {
           flex: 0 0 auto;
         }
-        > .o_content {
+        .o_content {
```

Comment:
these rules should only apply to the main view, not in nested views, so I think this should be kept. Of course, it doesn't work since you added a div around the form view, but I would try to find another workaround